source/guides/guides/continuous-integration.md

```diff
@@ -700,6 +700,38 @@ See {% url bahmutov/yarn-cypress-cache https://github.com/bahmutov/yarn-cypress-
 
 If you are running long runs on Docker, you need to set the `ipc` to `host` mode. {% issue 350 'This issue' %} describes exactly what to do.
 
+In a Docker container, the default size of the `/dev/shm` shared memory space is 64MB. This is not typically enough to run Chrome and can cause the browser to crash. You can fix this by passing the `--disable-dev-shm-usage` flag to Chrome with the following workarounds:
+
+### Cypress versions 4.0.0 and up
+
+```javascript
+module.exports = (on, config) => {
+  on('before:browser:launch', (browser = {}, launchOptions) => {
+    if (browser.family === 'chrome') {
+      console.log('Adding --disable-dev-shm-usage...')
+      launchOptions.args.push('--disable-dev-shm-usage')
+    }
+
+    return launchOptions
+  })
+}
+```
+
+### Cypress versions pre-4.0.0
+
+```javascript
+module.exports = (on, config) => {
+  on('before:browser:launch', (browser = {}, args) => {
+    if (browser.family === 'chrome') {
+      console.log('Adding --disable-dev-shm-usage...')
+      args.push('--disable-dev-shm-usage')
+    }
+
+    return args
+  })
+}
+```
```

Comment:
I would not document anything in the docs that's pre-current version, it just invites a precedent that we do not intend to maintain (unless we explicitly create versioned docs one day). This is reserved for the migration guide + history tabs + changelog.
```suggestion
```